playbooks/grafana.yaml

```
@@ -0,0 +1,51 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Set up and configure grafana.
+#
+#   - If a local ldap server is available,
+#     it will be configured as default auth method.
+---
+- name: Set up and configure grafana
+  hosts: grafana_servers
+  become: true
+
+  tasks:
+    - name: Execute apt role for sources configuration
+      ansible.builtin.include_role:
+        name: tina_pm.common.apt
+
+    - name: Install package
+      ansible.builtin.package:
+        name: grafana
+        state: present
+
+    - name: Configure grafana
+      ansible.builtin.template:
+        src: files/grafana/grafana.ini.j2
+        dest: /etc/grafana/grafana.ini
+        owner: root
+        group: grafana
+        mode: '0640'
+      notify: Restart grafana
```

```suggestion
      notify: _tina_restart_grafana
```

(this is just the convention we use in tina_pm.common)